consumers.go

```diff
@@ -368,6 +368,14 @@ func DeliverHeadersOnly() ConsumerOption {
 	}
 }
 
+// DeliverHeadersOnlyEx configures the consumer to deliver headers only. If headersOnly is false, the consumer will deliver the full message.
+func DeliverHeadersOnlyEx(headersOnly bool) ConsumerOption {
```

Comment:
Why the Ex in the name?

Reply:
Sorry I see now. This results in 2 ways to achieve the same. 

So I think I would rather have a function something like DeliverBodies() or similar function rather than end up with 2 ways to set true

Reply:
Sorry, updated. I gave you the context in our Slack instead of here.